tests/karma.js

```diff
@@ -302,6 +302,338 @@ beforeAll(function (done) {
           }
         }
       }))
+    .then(() =>
+      provider.addInteraction({
+        uponReceiving: 'set user attribute of an existing user, attribute is allowed',
+        withRequest: {
+          method: 'PUT',
+          path: Pact.Matchers.regex({
+            matcher: '.*\\/ocs\\/v(1|2)\\.php\\/cloud\\/users\\/' + config.testUser,
+            generate: '/ocs/v1.php/cloud/users/' + config.testUser
+          }),
+          headers: validAuthHeaders,
+          body: 'key=email&value=asd%40a.com'
+        },
+        willRespondWith: {
+          status: 200,
+          headers: {
+            'Access-Control-Allow-Origin': origin,
+            'Content-Type': 'text/xml; charset=utf-8',
+            'Access-Control-Allow-Methods': 'GET,OPTIONS,POST,PUT,DELETE,MKCOL,PROPFIND,PATCH,PROPPATCH,REPORT'
+          },
+          body: '<?xml version="1.0"?>\n' +
+            '<ocs>\n' +
+            ' <meta>\n' +
+            '  <status>ok</status>\n' +
+            '  <statuscode>100</statuscode>\n' +
```

Comment:
The `matcher` has ocs v1 or v2. When a v2 request does come (some day), this will return status 100. But ocs v2 should return status 200. These sort of things can be refactored when we have incoming requests that need it.

Reply:
This has already happened in other places in the test code. We will discuss separately - issue #572